### PR TITLE
Do not display all files when livesync is executed

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
When the livesync command is executed we do not need to log info for each file that is transfered. The information should be displayed when the --watch flag is set and some file is changed.

### CODE REVIEW

File: services/livesync-service-base.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ let gaze = require("gaze");
 
 class LiveSyncServiceBase implements ILiveSyncServiceBase {
 	private fileHashes: IDictionary<string>;
+	private showFullLiveSyncInformation: boolean;
```
Comment (rosen-vladimirov) [inline services/livesync-service-base.ts]: just set it to false here:
```TypeScript
private showFullLiveSyncInformation: boolean = false;
```
And skip the change in the constructor.

**Comment by justcodebuilduser** [comment]
:broken_heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: services/livesync-service-base.ts [hunk 2]
```diff
@@ -245,7 +247,7 @@ class LiveSyncServiceBase implements ILiveSyncServiceBase {
 
 	private transferFiles(deviceAppData: Mobile.IDeviceAppData, localToDevicePaths: Mobile.ILocalToDevicePathData[], projectFilesPath: string, isFullSync: boolean): IFuture<void> {
 		return (() => {
-			this.logFilesSyncInformation(localToDevicePaths, "Transferring %s.");
+			this.logFilesSyncInformation(localToDevicePaths, "Transferring %s.", this.$logger.trace);
```
Comment (rosen-vladimirov) [inline services/livesync-service-base.ts]: Please add 
```TypeScript
this.$logger.info("Transferring project files...");
```
before this call, so the users will know what's going on.

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by rosen-vladimirov** [comment]
:+1:  after green build

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**